Cargo.toml

```diff
@@ -1,18 +1,21 @@
 [package]
-name = "mcp23017"
-version = "1.0.0"
-description = "A rust driver for the MCP23017 (16-Bit I2C I/O Expander with Serial Interface)"
-authors = ["Luca Zulian <lucagiuggia@gmail.com>"]
+name = "mcp230xx"
+version = "0.0.0"
```

Comment:
Is 0.0.0 a valid version?

Reply:
It'll be increased with `cargo release`.